examples/sim_params.json

```diff
@@ -0,0 +1,16 @@
+{
+  "list_of_channels_under_test": [1, 18, 37, 56, 75],
+  "raman_parameters": {
+    "flag_raman": true,
```

Comment:
I checked minutes from 2019-05-28 before I wrote this comment, so I know that there's a difference between "use Raman engine" and "turn on Raman amplification", but I cannot map the result of that discussion into this JSON file.

My understanding is that it's possible to "use the Raman engine" with no Raman pumps being active (right?). How do I do that in terms of input JSON files?

The way I understand it, Raman pumps are controlled by using a `RamanFiber` in the network topology **and** listing the pumps in that file as well.

- What happens when I put a `RamanFiber` with no pumps into the JSON network file?
- What happens when I use `raman_paramaters` in the sim input, but with `flag_raman` set to `False`?